controllers/toggle_components.go

```diff
@@ -86,6 +89,11 @@ func (r *MultiClusterEngineReconciler) ensureNoConsoleMCE(ctx context.Context, b
 
 	r.StatusManager.RemoveComponent(toggle.EnabledStatus(namespacedName))
 	r.StatusManager.AddComponent(toggle.DisabledStatus(namespacedName, []*unstructured.Unstructured{}))
+	if !ocpConsole {
```

Comment:
Hey, does this end up with multiple statuses for the console component? Might be fine, just trying to make sure I understand how it works

Reply:
I've addressed this now in https://github.com/stolostron/backplane-operator/pull/323 , you are correct it created 2 statuses, with that PR now it'll only make one